options/locale/locale_en-US.ini

```diff
@@ -829,13 +829,15 @@ pulls.tab_files = Files Changed
 pulls.reopen_to_merge = Please reopen this pull request to perform a merge.
 pulls.merged = Merged
 pulls.has_merged = The pull request has been merged.
+pulls.work_in_progress = "This is a work in progress. Remove the <strong>%s</strong> prefix from the title when it's ready"
```

Comment:
`This is a work in progress.` would better be `This Pull Request is marked as a work in progress.`